File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_Value_LanguagesService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_Value_LanguagesService : BaseService<Sys_Value_Languages, ISys_Value_LanguagesRepository>
    , ISys_Value_LanguagesService
    {
    private readonly ISys_Value_LanguagesRepository _Sys_Value_LanguagesRepository;
    public Sys_Value_LanguagesService(ISys_Value_LanguagesRepository Sys_Value_LanguagesRepository) : base(Sys_Value_LanguagesRepository)
    {
    _Sys_Value_LanguagesRepository = Sys_Value_LanguagesRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Value_Languages.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Value_Languages } from './Sys_Value_Languages.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_Value_LanguagesService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Value_Id', 'Language_Id', 'Key_Id', 'Key_Type', 'Key_Column', 'Key_Code', 'Key_Value'];

    List: Sys_Value_Languages[] | undefined;
    ListFilter: Sys_Value_Languages[] | undefined;
    FormData!: Sys_Value_Languages;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Value_Languages";
    }
}

